EapWebInterface/src/test/java/de/hpi/unicorn/application/pages/input/generator/EventGeneratorTest.java

```diff
@@ -29,35 +30,39 @@
     private TypeTreeNode attribute = new TypeTreeNode(attributeName);
     private AttributeTypeTree attributeTree = new AttributeTypeTree(attribute);
     private EapEventType eventType = new EapEventType("TestType", attributeTree);
-    private HashMap<TypeTreeNode, String> attributeSchemas = new HashMap<>();
+    private List<AttributeInput> attributeInputs = new ArrayList<>();
     private ArrayList<EapEventType> eventTypes = new ArrayList<EapEventType>();
     private static final DateFormat dateFormatter = new SimpleDateFormat("yyyy/MM/dd'T'HH:mm");
 
+    static final Logger logger = Logger.getLogger(EventGeneratorTest.class);
+
     @Before
     public void setUp() {
         Persistor.useTestEnvironment();
         EapEvent.removeAll();
         EapEventType.removeAll();
-        attributeSchemas.put(attribute, attributeValue);
+        AttributeInput attributeInput = AttributeInput.attributeInputFactory(attribute);
+        attributeInput.setInput(attributeValue);
```

Comment:
It would be clever to have a constuctor that concats these two calls, right? I already used it like two times in my code.